src/lib/common/Kconfig

```diff
@@ -81,11 +81,43 @@ config DEFAULT_LOG_LEVEL_DEBUG
 	bool "debug"
 endchoice
 
-config KDBUS
-	bool "Kdbus"
-	depends on SOL_PLATFORM_LINUX && HAVE_SYSTEMD && HAVE_KDBUS
+config SOL_BUS
+	bool "Sol-bus"
+	depends on SOL_PLATFORM_LINUX && HAVE_SYSTEMD && HAVE_SD_BUS
 	default n
 
+menu "Hardware support"
+config HARDWARE_PWM
+	bool "pwm"
+	help
+          Globaly enables pwm hardware support.
+	default y
+
+config HARDWARE_SPI
+	bool "spi"
+	help
+          Globaly enables spi hardware support.
+	default y
+
+config HARDWARE_UART
+	bool "uart"
+	help
+          Globaly enables uart hardware support.
+	default y
+
+config HARDWARE_I2C
+	bool "i2c"
+	help
+          Globaly enables i2c hardware support.
+	default y
+
+config HARDWARE_GPIO
+	bool "gpio"
+	help
+          Globaly enables gpio hardware support.
+	default y
+endmenu
+
```

Comment:
I think you can send the fix with the modifications you will have to do
when check for GPIO on pin mux.

On Wed, Jul 1, 2015 at 12:39 PM, Leandro Dorileo <notifications@github.com>
wrote:

> In src/lib/common/Kconfig
> <https://github.com/solettaproject/soletta/pull/167#discussion_r33691401>:
>
> > +          Globaly enables uart hardware support.
> > +	default y
> > +
> > +config HARDWARE_I2C
> > +	bool "i2c"
> > +	help
> > +          Globaly enables i2c hardware support.
> > +	default y
> > +
> > +config HARDWARE_GPIO
> > +	bool "gpio"
> > +	help
> > +          Globaly enables gpio hardware support.
> > +	default y
> > +endmenu
> > +
>
> @mbelluzzo <https://github.com/mbelluzzo> again, I saw you introduced the
> HAVE_PIN_MUX configuration, I didn't see it before it got applied, but I
> would send a patch renaming this config to avoid future mixing (like I
> mentioned before) and the pattern propagation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/167/files#r33691401>.
>